### PR TITLE
Prefixes control classes and ID's with parent id

### PR DESCRIPTION
**Pull request by Cosrnos**
Fixes an issue where multiple instances of videojs would cause dom conflicts (#116)

### CODE REVIEW

File: src/videojs.ima.js [hunk 1]
```diff
@@ -46,6 +46,39 @@
...
+     * @param element
+     * @param controlName
+     * @private
+     */
+    player.ima.assignControlAttributes_ = function(element, controlName) {
+      element.id = controlPrefix + controlName;
+      element.className = controlPrefix + controlName + ' ' + controlName;
+    };
+
+    player.ima.getClassRegexp_ = function(className){
```
Comment (shawnbuso) [inline src/videojs.ima.js]: Can we add docs to these 3 new methods for posterity? Thanks!

**Comment by shawnbuso** [comment]
This is awesome, thanks! Just one small request to document a few new methods and we should be good to go.

**Comment by Cosrnos** [comment]
@shawnbuso Updated, though github isn't showing the updated diff in the discussion section. Let me know if you have any other changes you want me to make!

**Comment by shawnbuso** [comment]
Perfect, thanks!

**Pull request merged**